Makefile

```diff
@@ -291,6 +291,18 @@ bundle-community-k8s: bundle-community ## Generate bundle manifests and metadata
 bundle-community: bundle ##Add Community Edition suffix to operator name
 	sed -r -i "s|displayName: Self Node Remediation Operator.*|displayName: Self Node Remediation Operator - Community Edition|;" ${BUNDLE_CSV}
 
+# Split the VERSION into major, minor, and patch parts
+MAJOR := $(word 1, $(subst ., ,$(VERSION)))
+MINOR := $(word 2, $(subst ., ,$(VERSION)))
+PATCH := $(word 3, $(subst ., ,$(VERSION)))
+
+# Subtract 1 from the MINOR part
```

Comment:
I guess it depends:
For minor releases (patch version is 0) this is correct
For patch releases (patch version is >0) we need to decrease the patch version

Reply:
I think it's even more complicated, and automating it would require much more work: finding the actual previous release.
Reason: when releasing e.g. 0.6.0, the replace field needs to point to the last release. There is a good chance that the last one is 0.5.0 indeed, but it can also be any other patch version 0.5.x 🤷🏼‍♂️

Reply:
>but it can also be any other patch version 0.5.x 🤷🏼‍♂️

indeed this is a good point.

How about passing PREV_VERSION as you suggested, but in case it's not passed default to current behavior ?

Reply:
IMHO that would be an invite to forget setting it when needed...? 🤔

Reply:
I admit that this thought crossed my mind as well, I think it's a trade off between comfort and security. 
IMO we already have strong gateways around security so I think it's ok to lean the other way here. 
However it's not something I feel strongly about so if you prefer it the other way I'm fine with changing it.

Reply:
I'm not sure about it.
@clobrano @razo7 WDYT?

Reply:
I am ok with the new automation for the old version. But it is a step/issue that we should remember.

Reply:
Can we rely on git for this? The repo already knows what's the previous tag without the need to infer it from the current `VERSION`

Reply:
I was a bit optimistic. Git doesn't really know what is the previous version, just the _last_ one, so some use cases will still remain uncovered.

My question is, if `PREV_VERSION` is not provided by us explicitly and we don't have an automatic way to get it always right, how can we instruct the CI what version to use?

Reply:
In case it's not provided we deduct it from the current version (i.e either 0.X-1.0 or 0.X.Y-1), but since this cover only common case and not every case we can also provide PREV_VERSION explicitly.

Reply:
To be honest, I'd rather keep it mandatory. We still need to think if the replaced version will fall into the cases covered by the automation, so at this point just write it in the CLI.